### DISCUSSION TITLE
Uploading Existing Annotations to Train MAIA

### DISCUSSION THREAD

**Discussion by gnomelv** [biigle/core#969] [Q&A]
Hi everyone, I have a fairly large dataset of coral images and corresponding polygon coordinates marking the coral's location within the images. I was wondering if I could upload those to BIIGLE. The goal isn't necessarily to train an MAIA program that will classify the different coral species or types but to just mark what is coral in a given image. Any help would be much appreciated.

**Comment by mzur** [comment]
Hi, BIIGLE supports annotation import via COCO or iFDO files or via the API. In your case, COCO may be easiest. You can generate a COCO report from existing annotations in BIIGLE to get an example file.

Before I continue with how you can use the annotations for MAIA I have to tell you about the limitations. MAIA currently can't do automatic classification at all. The classification is done manually in the last step of MAIA. It does offer a very efficient user interface with smart sorting, though. In the future, it should also support classification (https://github.com/biigle/maia/issues/48).

Also, MAIA only creates circle annotations. So you can't get detailed polygons/segmentations with MAIA. This will also change at some point (https://github.com/biigle/maia/issues/97). If you plan to do a large-scale analysis we might be able to provide you with our experimental script that converts points/circles to polygons using SAM. A colleague is also currently working on an implementation for point to polygon conversion in BIIGLE.

MAIA allows you to use training annotations from the same volume or from a different volume. However, at the moment a different volume is only possible if the source volume (with annotations) and the target volume (without annotations) have image area metadata (this will change at some point https://github.com/biigle/maia/issues/126). If you don't have meaningful area information you can also fake it as a workaround. You can upload the image metadata as [CSV file](https://biigle.de/manual/tutorials/volumes/file-metadata) and just set the area of each image to 1.
